src/elements/profiles/Profiles.wc.svelte

```diff
@@ -147,15 +147,15 @@
         style="display: flex; justify-content: space-between; align-items: center;"
       >
         <h4 class="is-size-4">Profile Manager</h4>
-        <p>
+        <!-- <p>
```

Comment:
We can remove this paragraph if it's not needed anymore?

Reply:
done.

Reply:
added.